packages/react-tracking-pixel/src/TrackingPixel.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import isEqual from 'lodash/isEqual';
+import Preconnect from '@shopify/react-preconnect';
+import * as styles from './TrackingPixel.scss';
```

Comment:
Does this work?  In the `dist` code, I see a `require` for this, but no generated file.

What's the convention for adding CSS in quilt?  Can we just inline the style in this exceptional case?

Reply:
It seems to be causing some problems with the tests as well. Might be a better idea to use inline styles in this case.

Reply:
Also, there doesn't seem to be a convention for css in quilt (there are no other css/scss files in the project)... Seems like the convention is not to have any.